### DISCUSSION TITLE
This application is helpful, Thanks

### DISCUSSION THREAD

**Discussion by vipul1999** [ankurk91/google-chat-electron#25] [General]
Currently, on my ubuntu machine, I only get notifications of google chat till I keep the chrome running, So I needed the notifications of google chat even after I close the chrome. A feature like slack runs in the background even after you close it.

Tried all over the internet found nothing, then this is the application that saved my day,

So Thanks

**Comment by ankurk91** [comment]
@vipul1999 Thanks for the appropriation :smile:

**Comment by ankurk91** [comment]
I have rebuild this app in tauri now :)
https://github.com/ankurk91/google-chat-tauri
